README.md

```diff
@@ -1,159 +1,48 @@
-# ScopeLift Foundry Template
+# Flexible Voting Clients 
 
-An opinionated template for [Foundry](https://github.com/foundry-rs/foundry) projects.
+[Flexible Voting](https://github.com/ScopeLift/flexible-voting) is an extension to the widely used OpenZeppelin DAO Governor that enables novel voting patterns for delegates. Clients are contracts that leverage Flexible Voting to allow they these novel voting patterns, such as:
 
-_**Please read the full README before using this template.**_
 
-- [Usage](#usage)
-- [Overview](#overview)
-  - [`foundry.toml`](#foundrytoml)
-  - [CI](#ci)
-  - [Test Structure](#test-structure)
-- [Configuration](#configuration)
-  - [Coverage](#coverage)
-  - [Slither](#slither)
-  - [GitHub Code Scanning](#github-code-scanning)
+  - Voting with tokens while earning yield in DeFi
+  - Voting with tokens bridged to L2
+  - Shielded voting (i.e. secret/private voting)
+  - Cheaper subsidized signature based voting
+  - Better voting options with tokens held by custodians
```

Comment:
What does this last bullet mean?

Reply:
https://github.com/ScopeLift/flexible-voting-clients/pull/2/commits/373634678e35211a826b30c72694a589aef0fe8a